Kconfig.zephyr

```diff
@@ -949,6 +949,8 @@ config DEPRECATED
 	help
 	  Symbol that must be selected by a feature or module if it is
 	  considered to be deprecated.
+	  When adding this to an option, remember to add a mention in the GitHub issue tracking the
+	  removal of deprecated APIs/options: https://github.com/zephyrproject-rtos/zephyr/issues/76943
```

Comment:
This creates now 4 places to go update at release time.  It's feeling more like we should just have a "deprecation_process.rst" that all of these can link to.